oci/private/push.bzl

```diff
@@ -128,15 +141,19 @@ _attrs = {
         default = "push.sh.tpl",
         allow_single_file = True,
     ),
+    "_tools": attr.label(
+        cfg = _transition_to_target,
+        default = "oci_push_tools",
+    ),
```

Comment:
Minor: Fwiw, I think we can use the existing `:current_toolchain` targets here, and avoid the new `oci_push_tools` target.  I think the result is a bit more understandable, at the cost of having to declare crane and yq separately:

ie:
```starlark
  "_crane": attr.label(
      cfg = _transition_to_target,
      default = "@oci_crane_toolchains//:current_toolchain",
  ),
```

... and then refer to it in `_impl()` below using: `crane = ctx.attr._crane[0][platform_common.ToolchainInfo]`

And similar for yq.

Reply:
Great! I wasn't aware of these targets.